src/anbox/dbus/sd_bus_helpers.h

```diff
@@ -27,46 +27,28 @@ namespace sdbus {
 namespace vtable {
 constexpr sd_bus_vtable start(uint64_t flags)
 {
-  sd_bus_vtable v{};
-  v.type = _SD_BUS_VTABLE_START;
-  v.flags = flags;
-  v.x.start = decltype(v.x.start){sizeof(sd_bus_vtable)};
-
-  return v;
+  return SD_BUS_VTABLE_START(flags);
 }
 
 constexpr sd_bus_vtable end() {
-  sd_bus_vtable v{};
-  v.type = _SD_BUS_VTABLE_END;
-
-  return v;
+  return SD_BUS_VTABLE_END;
 }
 
 constexpr sd_bus_vtable method_o(const char* member, const char* signature,
                                  const char* result,
                                  sd_bus_message_handler_t handler, size_t offset,
                                  uint64_t flags) {
-  sd_bus_vtable v{};
-  v.type = _SD_BUS_VTABLE_METHOD;
-  v.flags = flags;
-  v.x.method = decltype(v.x.method){member, signature, result, handler, offset};
-
-  return v;
+  return sd_bus_vtable SD_BUS_METHOD_WITH_OFFSET(member, signature, result, handler, offset, flags);
 }
 
 constexpr sd_bus_vtable method(const char* member, const char* signature,
                                const char* result, sd_bus_message_handler_t handler,
                                uint64_t flags) {
-    return method_o(member, signature, result, handler, 0, flags);
+  return method_o(member, signature, result, handler, 0, flags);
 }
 
 constexpr sd_bus_vtable signal(const char* member, const char* signature, uint64_t flags) {
-  sd_bus_vtable v{};
-  v.type = _SD_BUS_VTABLE_SIGNAL;
-  v.flags = flags;
-  v.x.signal = decltype(v.x.signal){member, signature};
-
-  return v;
+  return sd_bus_vtable SD_BUS_SIGNAL(member, signature, flags);
 }
 
 constexpr sd_bus_vtable property(const char* member, const char* signature,
```

Comment:
The same thing should be done for the `property` methods too; even if it works for now, it might break in the future.

Reply:
Sure. I just was not certain if you would like my fix or if it would get a green build.

Reply:
Well, I am not a maintainer, I've just hit the same bug and when I was about to submit a PR, I saw yours, and I thought it's worth the hint :).

Reply:
It is, thank you! I finished the fix accordingly.